modules/cloudtrail/main.tf

```diff
@@ -78,7 +83,7 @@ resource "aws_cloudwatch_log_group" "cloudtrail" {
   tags       = var.tags
 }
 
-resource "aws_cloudwatch_log_stream" "cloudtrail_stream" {
+resource "aws_cloudwatch_log_stream" "cloudtrail-stream" {
```

Comment:
I had to look at this 10 times before I spotted the difference 😂

Reply:
😆 I realised I used dashes everywhere else, it took me a while to realise why Terraform was telling me it'll destroy and recreate the same resource!